src/guide/scaling-up/state-management.md

```diff
@@ -151,9 +151,9 @@ export default {
 
 </div>
 
-Agora sempre que o objeto `store` é alterado, ambos `<ComponentA>` e `<ComponentB>` atualizarão suas apresentações automaticamente - agora nós temos uma fonte única de verdade.
+Agora sempre que o objeto `store` é alterado, ambos `<ComponentA>` e `<ComponentB>` atualizarão suas apresentações automaticamente - agora nós teremos uma fonte única de verdade.
```

Comment:
Em inglês:
_Now whenever the `store` object is mutated, both `<ComponentA>` and `<ComponentB>` will update their views automatically - we have a single source of truth now._

Sugestão:
```suggestion
Agora sempre que o objeto `store` é alterado, ambos `<ComponentA>` e `<ComponentB>` atualizarão suas apresentações automaticamente - agora temos uma fonte única de verdade.
```

E talvez trocar **apresentações** por **visualizações**, o que dizem?

Reply:
Acredito que seria mais interesante, uma palavra também que sugiro seria 'Equipa', por que não mudara para 'Time' então?